babel-code-frame/babel-code-frame-tests.ts

```diff
@@ -0,0 +1,33 @@
+import codeFrame from "babel-code-frame";
+import {expect} from 'chai';
+
+describe('babel-code-frame', () => {
+    it('frames given code', () => {
+        const code = toTrimmedLines(`
+            const number = 1;
+            var string = 'foo';
+
+            function print(name: string) {
+                console.log(string + name);
+            }
+        `).join('\n');
+
+        /** Convert output lines to an array of trimmed lines for easier comparing */
+        function toTrimmedLines(lines: string): string[] {
+            return lines.split("\n").map((line) => line.trim());
+        }
+
+        const framed = toTrimmedLines(codeFrame(code, 5, 22, { forceColor: true }));
```

Comment:
You don't even need the chai overhead. Just
```ts
import codeFrame from "babel-code-frame";
codeFrame("", 5, 22, {
    forceColor: true,
    ...test out other options...
})
```
Also, is the options object optional? Could include a test `babelCodeFrame("", 0, 0)`.
```